modules/gitops-deleting-rolloutmanager-custom-resource.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/argo-rollouts-for-progressive-delivery-of-deployments.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-deleting-rolloutmanager-custom-resource_{context}"]
+= Deleting a RolloutManager custom resource
+
+Uninstalling the {gitops-title} Operator does not remove the resources that were created during installation. To uninstall the progressive delivery of deployments by using {gitops-title}, you must explicitly delete the resources by deleting the `RolloutManager` custom resource (CR), and then uninstall the {gitops-title} Operator.
+
+.Prerequisites
+
+* {gitops-title} 1.9.0 or a newer version is installed in your cluster.
+* A `RolloutManager` CR exists in your namespace.
+
+.Procedure 
+
+. Log in to the {product-title} web console as a cluster administrator. 
+
+. In the *Administrator* perspective, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where `RolloutManager` CR for the progressive delivery is setup from the *Project* drop-down menu.
+
+. Select *OpenShift GitOps Operator* from the installed operators.
+
+. In the Tabs section, click *RolloutManager* to find the list of RolloutManager instances under the *RolloutManagers* section.
+
+. Select the instance by clicking on its name. 
+
+. At the top right corner of the pane, click *Actions* -> *Delete RolloutManager* from the drop-down menu, and click *Delete* to confirm in the dialog box.
+
+. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the RolloutManager instance is not available anymore.
+
+. In the left navigation pane, verify the deletion of the namespace-scoped supporting resources:
++ 
+* Click *Workloads* -> *Deployments* and ensure the `argo-rollouts` deployment is deleted.
+* Click *Workloads* -> *Secrets* and ensure the `argo-rollouts-notification-secret` secret is deleted.
+* Click *Networking* -> *Services* and ensure the `argo-rollouts-metrics` service is deleted.
+* Click *User Management* -> *Roles* and ensure the `argo-rollouts` role and `argo-rollouts-aggregate-to-admin`, `argo-rollouts-aggregate-to-edit`, and `argo-rollouts-aggregate-to-view` cluster roles are deleted.
+* Click *User Management* -> *RoleBindings* and ensure the `argo-rollouts` role binding is deleted.
```

Comment:
My comments from modules/gitops-creating-rolloutmanager-custom-resource.adoc also apply to this section